cves/CVE-2013-1665.yml

```diff
@@ -146,8 +164,14 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    This is an XML external entity attack. Python documentation even states that
+    the issue is nothing new. Billion laughs, an XML external entity attack,
+    has been reported since 2003. It's plausible that an automated tool
+    could have been developed after several such similar reports to discover these
+    security loopholes. The tool can also get its forbidden entity libraries updated
+    overtime for more rigorous vulnerability discoveries in the long run.
+  answer: true
```

Comment:
Be sure to implement sources

Reply:
I will put on the website that I look at here.